online-shop/src/app/modules/shared/components/presentational/icon-button/icon-button.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  standalone:true,
+  selector: 'app-icon-button',
+  templateUrl: './icon-button.component.html',
+  styleUrls: ['./icon-button.component.scss']
+})
+export class IconButtonComponent {
+  @Input() iconType:string | undefined;
+  @Output() clickEvent: EventEmitter<void> = new EventEmitter<void>()
+
+  constructor(private router: Router) {}
+
+  handleClick(): void {
+    // if(this.iconType=="Shop"){
```

Comment:
Commented code should not appear in Pull Requests, please remove this, as well as the router if you're not going to use it.